Abcbadq/sjsu_shop
# Sjsu 1017, add recipe to data base

What i did 
_ Populated recipe data in profile, recipe database
_ create MyRecipe.js where user can go and create recipe

How to test
_ go to user profile -> view recipe -> createRecipe 
_ go to MLab and check the related profile and recipe database to see if the recipe is added
